ucm2/Intel/sof-essx8336/Hdmi.conf

```diff
@@ -0,0 +1,62 @@
+SectionDevice."HDMI1" {
```

Comment:
Please, use the codecs/hda/hdmi.conf template like HDA/Hdmi.conf does. It will resolve @yangxiaohua2009 issue as well.